modules/rosa-cluster-autoscaler-cli-interactive-after.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-autoscale-cli-interactive_after_{context}"]
+= Enable autoscaling after cluster creation by using the interactive mode with the ROSA CLI
+
+You can use the installer interactive mode to set cluster-wide autoscaling behavior after cluster creation.
+
+.Procedure
+
+. After you have created a cluster, type the following command.
++
+[source,terminal]
+----
+$ rosa create autoscaler --cluster=<mycluster> -interactive
```

Comment:
@osherdp Is this command correct, based on your comment? Does this command only apply to non-interactive shells?

Reply:
``rosa create autoscaler --cluster=<mycluster> --interactive`` applies only to interactive shells
``rosa create autoscaler --cluster=<mycluster> <parameters and without --interactive>`` also applies to non-interactive shells
I'd suggest giving it a try so you can make sure the commands are working (for example, in lots of places you have ``-interactive`` instead of ``--interactive`` which would have caught when trying to run it)